spec/workflow/states/wait_spec.rb

```diff
@@ -0,0 +1,22 @@
+RSpec.describe Floe::Workflow::States::Pass do
+  let(:workflow) { Floe::Workflow.load(GEM_ROOT.join("examples/workflow.asl")) }
+  let(:state)    { workflow.states_by_name["WaitState"] }
+
+  describe "#run!" do
+    it "sets the result to the result path" do
```

Comment:
The test isn't showing this (or is this WIP?)

Reply:
Fixed